src/privatesend/privatesend-client.cpp

```diff
@@ -16,15 +16,11 @@
 #include "util.h"
 #include "utilmoneystr.h"
 #include "validation.h"
+#include "wallet/wallet.h"
 #include "wallet/coincontrol.h"
 
 #include <memory>
 
-#ifdef ENABLE_WALLET
```

Comment:
You probably still need the ENABLE_WALLET around the include

Reply:
resolved